packages/wrangler/src/deploy/deploy.ts

```diff
@@ -484,6 +485,20 @@ export default async function deploy(props: Props): Promise<{
 		}
 	}
 
+	if (isInteractive() || props.strict) {
+		const remoteSecretsCheck = await checkRemoteSecretsOverride(
+			config,
+			props.env
+		);
+
+		if (remoteSecretsCheck?.override) {
+			logger.warn(remoteSecretsCheck.deployErrorMessage);
+			if (!(await deployConfirm("Would you like to continue?"))) {
+				return { versionId, workerTag };
+			}
+		}
+	}
```

Comment:
Is this right? I think this will still result in strict mode deploying in CI, when I think it should fail

Reply:
Amazing!